app/helpers/katello/subscription_mailer_helper.rb

```diff
@@ -0,0 +1,18 @@
+module Katello
+  module SubscriptionMailerHelper
+    include Rails.application.routes.url_helpers
+
+    def report_url
+      base_url = report_data_report_template_path(@report_template.id, job_id: @provider_job_id) unless @pools.empty?
+      "#{Setting[:foreman_url]}#{base_url}"
+    end
+
+    def start_report_task(mail_query)
+      @report_template ||= ReportTemplate.find_by(name: "Subscription - Entitlement Report")
+      template_input_id = TemplateInput.find_by_name("Days from Now").id.to_s
+      params = { format: 'csv', template_id: @report_template.id, input_values: { template_input_id => { value: mail_query} } }
+      composer = ReportComposer.new(params)
```

Comment:
The params look good to me.  However, when I go to the report page in the web UI, I'm seeing the following:
![days_from_now_error](https://user-images.githubusercontent.com/22042343/80648458-329d5a80-8a3e-11ea-92b0-d016e3bf1cf5.png)
Error stack: http://pastebin.test.redhat.com/860473

Reply:
update: We're correctly finding the `template_input_id` now, so this error is resolved. :partying_face: